Jenkinsfile

```diff
@@ -98,7 +98,7 @@ pipeline {
                 stage ('Linux') {
                     steps {
                         // This timeout was reduced from 3 hours after this stage was modified to no longer run the itests. In the future, we may want to reduce it further if there is confidence that the build will finish faster.
```

Comment:
❔ should we update the comment? It isn't reduced from 3 hours anymore.

Reply:
@shaundmorris sounds good, I'll remove the comment

Reply:
@shaundmorris all set